go/mysql/collations/8bit.go

```diff
@@ -113,6 +115,17 @@ func (c *Collation_8bit_bin) Wildcard(pat []byte, matchOne rune, matchMany rune,
 	return newEightbitWildcardMatcher(&sortOrderIdentity, c.Collate, pat, matchOne, matchMany, escape)
 }
 
+func (c *Collation_8bit_bin) GetLowerTable() *[256]byte {
+	for _, c := range c.simpletables.tolower {
+		fmt.Println(c)
+	}
+	return c.simpletables.tolower
+}
+
+func (c *Collation_8bit_bin) GetUpperTable() *[256]byte {
+	return c.simpletables.toupper
+}
+
```

Comment:
You need to implement `ToUpper` and `ToLower` methods in the `Collation` interface. Exposing the lower/upper tables directly won't work because other collations (such as UTF8) do not use tables for upcasing.